base/complex.jl

```diff
@@ -1100,6 +1100,11 @@ function round(z::Complex, rr::RoundingMode=RoundNearest, ri::RoundingMode=rr; k
             round(imag(z), ri; kwargs...))
 end
 
+function round(::Type{Complex{T}}, z::Complex) where {T}
```

Comment:
Shouldn't this take an optional `r::RoundingMode` argument?

(That starts to get into the question of defining [complex floor](https://www.jsoftware.com/papers/eem/complexfloor.htm) from #42060?)